Sources/CollectionView/CollectionAdapter.swift

```diff
@@ -0,0 +1,154 @@
+//
+//  CollectionAdapter.swift
+//  ETDataDrivenViewKit-iOS
+//
+//  Created by Růžička Jakub on 16/11/2018.
+//  Copyright © 2018 Etnetera a. s. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+open class CollectionAdapter: NSObject {
+    
+    // MARK: - Variables
+    // MARK: public
+    
+    public var data: [DiffableType] = [] {
+        didSet {
+            //TODO:
```

Comment:
Proč je tady TODO? Já jen jestli jsi tady na něco nezapomněl.

Reply:
Doplnil jsem pouze komentář, zatím to stačí takhle